editor.planx.uk/src/@planx/components/MapAndLabel/model.ts

```diff
@@ -0,0 +1,12 @@
+import { MoreInformation, parseMoreInformation } from "../shared";
+
+export interface MapAndLabel extends MoreInformation {
+  fn: string;
```

Comment:
At minimum, every component also has a `title` & `description` available to configure in the Editor - I'd expect to see those fields here !